libraries/core_libs/consensus/src/dag/dag.cpp

```diff
@@ -542,6 +542,12 @@ uint DagManager::setDagBlockOrder(blk_hash_t const &new_anchor, uint64_t period,
     return 0;
   }
 
+  if (new_anchor == blk_hash_t(0)) {
+    period_ = period;
+    LOG(log_nf_) << "Set new period " << period << " with anchor " << new_anchor;
```

Comment:
`LOG(log_nf_) << "Set new period " << period << " with NULL_BLOCK_HASH anchor;` would be probably easier to notice when going through logs

Reply:
Fixed, thanks